single-campaigns.php

```diff
@@ -0,0 +1,90 @@
+$post->set_issues_links();
+
+// Get the cmb2 custom fields data
+// Articles block parameters to populate the articles block
+// p4_take_action_page parameter to populate the take action boxout block
+// Author override parameter. If this is set then the author profile section will not be displayed.
+$page_meta_data                 = get_post_meta( $post->ID );
+$campaign_template              = ! empty( $page_meta_data['_campaign_page_template'][0] ) ? $page_meta_data['_campaign_page_template'][0] : 'antarctic';
+$page_terms_data                = get_the_terms( $post, 'p4-page-type' );
+$background_image_id                = get_post_meta( get_the_ID(), 'background_image_id', 1 );
+$context['background_image']        = wp_get_attachment_url( $background_image_id );
+$context['background_image_srcset'] = wp_get_attachment_image_srcset( $background_image_id, 'full' );$take_action_page               = $page_meta_data['p4_take_action_page'][0] ?? '';
+$context['header_title']        = is_front_page() ? ( $page_meta_data['p4_title'][0] ?? '' ) : ( $page_meta_data['p4_title'][0] ?? $post->title );
+$context['header_subtitle']     = $page_meta_data['p4_subtitle'][0] ?? '';
+$context['header_description']  = wpautop( $page_meta_data['p4_description'][0] ?? '' );
+$context['header_button_title'] = $page_meta_data['p4_button_title'][0] ?? '';
+$context['header_button_link']  = $page_meta_data['p4_button_link'][0] ?? '';
+$context['page_type']           = $page_terms_data[0]->name ?? '';
+$context['page_term_id']        = $page_terms_data[0]->term_id ?? '';
+$context['page_category']       = $category->name ?? __( 'Post page', 'planet4-master-theme' );
+$context['page_type_slug']      = $page_terms_data[0]->slug ?? '';
+$context['social_accounts']     = $post->get_social_accounts( $context['footer_social_menu'] );
+$context['og_title']            = $post->get_og_title();
+$context['og_description']      = $post->get_og_description();
+$context['og_image_data']       = $post->get_og_image();
+$context['custom_body_classes'] = 'brown-bg theme-' . $campaign_template;
+
+$context['filter_url'] = add_query_arg( [
+	's'                                       => ' ',
+	'orderby'                                 => 'relevant',
+	'f[ptype][' . $context['page_type'] . ']' => $context['page_term_id'],
+], get_home_url()
+);
+
+
+// Build the shortcode for articles block.
+if ( 'yes' === $post->include_articles ) {
+	$post->articles = "[shortcake_articles exclude_post_id='" . $post->ID . "' /]";
+}
+
+// Build the shortcode for take action boxout block
+// Break the content to retrieve first 2 paragraphs and split the content if the take action page has been defined.
+if ( ! empty( $take_action_page ) ) {
+	$post->take_action_page   = $take_action_page;
+	$post->take_action_boxout = "[shortcake_take_action_boxout take_action_page='$take_action_page' /]";
+}
+
+// Build an arguments array to customize WordPress comment form.
+$comments_args = [
```

Comment:
You can remove these lines, if the cpt won't support comments (although I see cpt supports comment).
https://github.com/greenpeace/planet4-master-theme/blob/3db348cdad85de31ce37b0e4da08987ae9b421b6/single-campaigns.php#L59-L82

Or use this in the single-campaigns twig template, if we are gonna use comments for this cpt
https://github.com/greenpeace/planet4-master-theme/blob/3db348cdad85de31ce37b0e4da08987ae9b421b6/templates/single.twig#L89

Reply:
Will do. Do we want comments on campaign pages?

Reply:
No, I don't think so. You can remove this whole block for the moment.